providers/fips/fipsprov.c

```diff
@@ -342,7 +344,7 @@ static const OSSL_ALGORITHM_CAPABLE fips_ciphers[] = {
          ossl_cipher_capable_aes_cbc_hmac_sha256),
     ALGC(PROV_NAMES_AES_256_CBC_HMAC_SHA256, ossl_aes256cbc_hmac_sha256_functions,
          ossl_cipher_capable_aes_cbc_hmac_sha256),
-#ifndef OPENSSL_NO_DES
+#if !defined (OPENSSL_NO_DES)
```

Comment:
?

Reply:
The original PR did not have this. I had a comment somewhere above saying that I should use the new form, so I changed them. I'm happy to withdraw the whole mess and let someone else fix this breakage. Please see my other comments with regards to configuration changes.

I can undo this whole mess easily enough but need a consistent message from the reviewers, please.

Reply:
Yes, please undo these changes.

Reply:
Done